src/TaskLineRenderer.ts

```diff
@@ -28,7 +28,7 @@ export const LayoutClasses: { [c in TaskLayoutComponent]: string } = {
     scheduledDate: 'task-scheduled',
     doneDate: 'task-done',
     recurrenceRule: 'task-recurring',
-    blockLink: '',
+    blockLink: 'block-link',
```

Comment:
Is there a particular reason for making this not begin with `task-`?

Reply:
No, it was just a spontaneous decision

Reply:
Thanks!